internal/provider/mailhog/mailhog.go

```diff
@@ -15,13 +15,14 @@ const (
 )
 
 // Send the email to Mailhog.
-func Send(msg *mail.Message) error {
+func Send(to []string, msg *mail.Message) error {
 	data, err := mailutils.MessageToBytes(msg)
 	if err != nil {
 		return err
 	}
+	if msgTo := msg.Header.Get(mailutils.HeaderTo); msgTo != "" {
```

Comment:
I wonder what this looks like if there is multiple `To` headers already present

Reply:
Should this use the same approach as https://github.com/skpr/mail/pull/6/files#diff-ccb37b233075effa36d68fe5de078faf1aaa24abf207b8b84be60b70b8b6d8d1R31